src/app/subject/components/source-graphs/source-graphs.component.html

```diff
@@ -18,35 +20,43 @@
 
     <div class="sensors">
       <ng-container *ngIf="source.sourceData">
-        <div class="sensor"
+        <div
+          class="sensor"
           *ngFor="let sourceData of source.sourceData; trackBy: trackBySourceDataUid"
         >
           <app-source-graph
             *ngIf="sourceData?.visible"
             [sourceData]="sourceData"
             [sensorData]="sensorsData[sourceData.uid]?.data"
             [isLoaded]="isDataLoaded[sourceData.uid]"
-            [sensorId]="sourceData.uid"></app-source-graph>
+            [sensorId]="sourceData.uid"
+          ></app-source-graph>
         </div>
       </ng-container>
       <ng-container *ngIf="source.sourceData === null">
         <div class="sensor empty message">No SourceData</div>
       </ng-container>
     </div>
-    <div class="line"
+    <div
+      class="line"
       [style.transform]="'translateX('+ lineX +'px)'"
-      [style.opacity]="tooltipShow"></div>
+      [style.opacity]="tooltipShow"
+    ></div>
   </div>
 </div>
 
-<!-- <div class="bottom-group">
+<div class="bottom-group">
   <app-source-date-axis
     class="common-axis"
-    [dates]="dates"
-  ></app-source-date-axis> -->
+    [timeFrame]="sensorsDataTimeFrame"
+  ></app-source-date-axis>
```

Comment:
LGTM. When I run it on my machine, I do not see this common-axis for data. I can see the volume bar. 
I see the `timeFrame` value is null.